C++/source/string/edit_distance.cpp

```diff
@@ -0,0 +1,45 @@
+/*
+
+Edit Distance Algorithm :-
+--------------------------------
+Description: Find the minimum number of operations required to convert one string into another.
+
+Time Complexity :- O(m * n)
+Space Complexity :- O(m * n)
+                    Where m = length of string1 and n = string2
+
+*/
+
+
+
+#include<bits/stdc++.h>
```

Comment:
It's good practice to only `#include` those headers that are required.

Also, this header is not present on OS X, as so the code does not compile on it.